google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/CloudApiManagementConfirmationDialog.java

```diff
@@ -174,9 +174,22 @@ Path getServiceAccountKeyDownloadPath() {
   @Override
   protected ValidationInfo doValidate() {
     if (createNewServiceAccountCheckbox.isSelected()) {
-      if (StringUtils.isEmpty(serviceAccountNameTextField.getText())) {
+      String name = serviceAccountNameTextField.getText().trim().toLowerCase();
```

Comment:
Nit: just call `getServiceAccountName()`?

Reply:
ah yeah. thats much better and safer

Reply:
done - also did the same for service account path